src/MoBi.Presentation/Tasks/Interaction/InitialConditionsTask.cs

```diff
@@ -130,6 +135,31 @@ public IMoBiCommand UpdateInitialConditionScaleDivisor(TBuildingBlock buildingBl
          return new UpdateInitialConditionScaleDivisorCommand(buildingBlock, initialCondition, newScaleDivisor, oldScaleDivisor).Run(Context);
       }
 
+      public void ExtendExpressionProfileInitialConditions(ExpressionProfileBuildingBlock buildingBlock)
+      {
+         var (spatialStructure, _) = SelectBuildingBlocksForExtend(moleculeRequired: false);
+         if (spatialStructure == null)
+            return;
+
+         var moleculeBuilder = _moleculeBuilderTask.CreateDefault(buildingBlock.MoleculeName);
+         var startFormula = defaultStartFormulaFrom(moleculeBuilder, buildingBlock);
+
+         var newStartValues = spatialStructure.PhysicalContainers.Select(container => _initialConditionsCreator.CreateInitialCondition(container, moleculeBuilder, startFormula)).ToList();
+
+         updateDefaultIsPresentToFalseForSpecificExtendedValues(newStartValues, buildingBlock.InitialConditions.ToList());
+         AddCommand(Extend(newStartValues, buildingBlock));
+      }
+
+      private IFormula defaultStartFormulaFrom(MoleculeBuilder moleculeBuilder, ExpressionProfileBuildingBlock buildingBlock)
+      {
+         return buildingBlock.InitialConditions.Any() ? mostUsedFormula(buildingBlock.InitialConditions) : moleculeBuilder.DefaultStartFormula;
```

Comment:
this is a bit weird. I would expect the DefaultStartFormula to be used if there is one and if not, we used the mostUsedOne. Here the code reads the opposite

Reply:
I think we decided on the opposite didn't we? 
That's what I understood yesterday. There will be a defaultStartFormula because that's the way the basic builder is configured a ConstantFormula with value 0.